content/en/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -28,52 +28,50 @@ Kubernetes Project {{< glossary_tooltip text="Contributors" term_id="contributor
+<!-- body -->
 
 ## Configuration
 
-*Configuration files* and *flags* are documented in the Reference section of the online
-documentation, under each binary:
+*Configuration files* and *command arguments* are documented in the [Reference](/docs/reference/) section of the online
+documentation, with a page for each binary:
 
-* [kubelet](/docs/reference/command-line-tools-reference/kubelet/)
-* [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/)
-* [kube-apiserver](/docs/reference/command-line-tools-reference/kube-apiserver/)
-* [kube-controller-manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
-* [kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/).
+* [`kube-apiserver`](/docs/reference/command-line-tools-reference/kube-apiserver/)
+* [`kube-controller-manager`](/docs/reference/command-line-tools-reference/kube-controller-manager/)
+* [`kube-scheduler`](/docs/reference/command-line-tools-reference/kube-scheduler/)
+* [`kubelet`](/docs/reference/command-line-tools-reference/kubelet/)
+* [`kube-proxy`](/docs/reference/command-line-tools-reference/kube-proxy/)
 
-Flags and configuration files may not always be changeable in a hosted Kubernetes service or a
+Command arguments and configuration files may not always be changeable in a hosted Kubernetes service or a
 distribution with managed installation. When they are changeable, they are usually only changeable
-by the cluster administrator. Also, they are subject to change in future Kubernetes versions, and
+by the cluster operator. Also, they are subject to change in future Kubernetes versions, and
 setting them may require restarting processes. For those reasons, they should be used only when
 there are no other options.
 
-*Built-in Policy APIs*, such as [ResourceQuota](/docs/concepts/policy/resource-quotas/),
-[PodSecurityPolicies](/docs/concepts/security/pod-security-policy/),
+Built-in *policy APIs*, such as [ResourceQuota](/docs/concepts/policy/resource-quotas/),
 [NetworkPolicy](/docs/concepts/services-networking/network-policies/) and Role-based Access Control
-([RBAC](/docs/reference/access-authn-authz/rbac/)), are built-in Kubernetes APIs.
-APIs are typically used with hosted Kubernetes services and with managed Kubernetes installations.
-They are declarative and use the same conventions as other Kubernetes resources like pods,
+([RBAC](/docs/reference/access-authn-authz/rbac/)), are built-in Kubernetes APIs that provide declaratively configured policy settings.
+APIs are typically usable even with hosted Kubernetes services and with managed Kubernetes installations.
+These declarative APIs follow the same conventions as other Kubernetes resources such as Pods,
 so new cluster configuration can be repeatable and be managed the same way as applications.
-And, where they are stable, they enjoy a
+And, where the APIs are stable, they enjoy a
```

Comment:
nit: And, where the APIs are stable, they *enjoy* a

Does this edit make sense:
The built-in policy APIs are stable and adhere to the Kubernetes API [defined support policy](/docs/reference/using-api/deprecation-policy).

Reply:
Not all built-in policy APIs are stable. For example, PodSecurityPolicy which is built-in but deprecated and won't graduate.